modules/mobile/spec/request/messages_request_spec.rb

```diff
@@ -137,6 +137,7 @@
           expect(response.body).to be_a(String)
           expect(JSON.parse(response.body)['data']['attributes']['subject']).to eq('CI Run')
           expect(JSON.parse(response.body)['data']['attributes']['body']).to eq('Continuous Integration')
+          expect(JSON.parse(response.body)['data']['attributes']['attachment_size']).to be_positive.or be_nil
```

Comment:
If you take a look at a sample response below, you'll see that the attachments are returned in the `included` array.  So this probably has to be something like `expect(JSON.parse(response.body)['included'][0]['attributes']['attachment_size']).to be_positive.or be_nil`
```
{
    "data": {
        "id": "1958827",
        "type": "messages",
        "attributes": {
            "message_id": 1958827,
            "category": "TEST_RESULTS",
            "subject": "Your lab results",
            "body": "test with attachment 2",
            "attachment": true,
            "sent_date": "2021-06-03T17:26:52.000Z",
            "sender_id": 523757,
            "sender_name": "FREEMAN, MELVIN  V",
            "recipient_id": 1918816,
            "recipient_name": "VA Flagship mobile applications interface 2_DAYT29",
            "read_receipt": null
        },
        "relationships": {
            "attachments": {
                "data": [
                    {
                        "id": "1958826",
                        "type": "attachments"
                    }
                ]
            }
        },
        "links": {
            "self": "http://2811b521c3f13d58f79b6ec3c8735f1e-api.review.vetsgov-internal/mobile/v0/messaging/health/messages/1958827"
        }
    },
    "included": [
        {
            "id": "1958826",
            "type": "attachments",
            "attributes": {
                "name": "bob.jpg",
                "message_id": 1958827,
            },
            "links": {
                "download": "http://2811b521c3f13d58f79b6ec3c8735f1e-api.review.vetsgov-internal/mobile/v0/messaging/health/messages/1958827/attachments/1958826"
            }
        }
    ]
}
```